crates/bevy_gltf/src/lib.rs

```diff
@@ -378,6 +379,11 @@ pub struct GltfMaterialExtras {
     pub value: String,
 }
 
+/// The material name of a gltf primitive.
```

Comment:
This should link to the glTF's spec's section on materials, and use the appropriate capitalization.